packages/fxa-auth-server/config/index.ts

```diff
@@ -830,7 +830,7 @@ const conf = convict({
     stripeTaxRatesCacheTtlSeconds: {
       doc: 'The number of seconds to cache tax rates from Stripe',
       format: 'int',
-      default: 600,
+      default: 64000000, // about a couple years
```

Comment:
I don't know if we have a value set for this in stage or prod so it'd be good to check.

Reply:
I checked, and we do not set this env var upstream in mozilla-services/cloudops-deployment, so the default should be the value in all environments.